clang/lib/Tooling/DependencyScanning/DependencyScanningTool.cpp

```diff
@@ -231,6 +243,39 @@ class IncludeTreePPConsumer : public PPIncludeActionsConsumer {
 };
 } // namespace
 
+void IncludeTreePPConsumer::initialize(CompilerInstance &CI) {
+  if (llvm::Error E = PrefixMapping.configurePrefixMapper(
+          CI.getInvocation(), PrefixMapper)) {
+    ErrorToReport = std::move(E);
```

Comment:
Have you considered having `initialize` return `Error` and have it produce a diagnostic in the scanner?

Reply:
Good idea, did it for `finalize` as well.